src/mutation/AddEventMutation.js

```diff
@@ -65,32 +72,38 @@ export default mutationWithClientMutationId({
     // @TODO improve validation logic
     if (!title.trim() || title.trim().length < 2) {
       return {
-        error: 'Título inválido',
+        message: 'Invalid title',
+        error: 'INVALID_TITLE',
       };
     }
 
     if (!description.trim() || description.trim().length < 2) {
       return {
-        error: 'Descrição inválida',
+        message: 'Invalid description',
+        error: 'INVALID_DESCRIPTION',
       };
     }
 
     // Create new record
     const data = new EventModel({
       ...args,
     });
-    const event = await data.save();
+    await data.save();
 
-    return event;
+    // return event;
+    return {
+      message: 'Event created with success',
+      error: null,
+    };
```

Comment:
we need to return the event :p

Reply:
yes but we need to put at the output fields

Reply:
@lXSPandora , @jaburcodes is right.
We should return an Event.
Maybe we should think of a list of events on client side, which represents a connection on server side, so we should return an Event Edge